UnityProject/Assets/Scripts/Clothing/FacehuggerImpregnation.cs

```diff
@@ -112,7 +112,7 @@ private IEnumerator Release()
 			//TODO wait until the object's velocity is 0 instead of a fixed amount of time!
 			yield return WaitFor.Seconds(0.6f);
 			Spawn.ServerPrefab(facehugger, gameObject.transform.position);
-			Despawn.ServerSingle(gameObject);
+			_ = Despawn.ServerSingle(gameObject);
```

Comment:
Why is this needed?

Reply:
to stop warning about the call not being awaited